ibm-messaging-mq-cloud-showcase-app/backend/AWS-showcase-backend/models/Responder.js

```diff
@@ -43,7 +43,9 @@ class Responder {
             })
             .catch((err) => {
                 debug_warn(`Responder ${this.myID} error obtaining message ${err}`);
```

Comment:
This catch is only going to be triggered when the get has failed. In which case another attempted get will also fail. 

When there is no reply to queue the code does a `resolve(null)` in which case the catch clause will not be triggered. 

Has this logic been tested?